
/voyager/voyager_simulation:
  ros__parameters:
    initial_conditions:
      position:
        x: 0.0
        y: 0.0
        yaw: 0.0
      velocity:
        surge: 0.0
        sway: 0.0
        yaw_rate: 0.0
    tf_prefix: voyager
    body_frame_id: base_link_ned
    world_frame_id: world

/drillship/drillship_simulation:
  ros__parameters:
    initial_conditions:
      position:
        x: 4.0
        y: 0.0
        yaw: 0.0
      velocity:
        surge: 0.0
        sway: 0.0
        yaw_rate: 0.0
    tf_prefix: drillship
    body_frame_id: base_link_ned
    world_frame_id: world